etc/node_modules/@rokid/core-cloud-logger/index.js

```diff
@@ -75,7 +75,7 @@ function getToken () {
     }
     var sign = makeSign(props)
     var authKey = authorization(Object.assign({}, props, { sign: sign }))
-    var env = profile.env === 'dev' || profile.env === 'test' ? '-dev': ''
+    var env = profile.env === 'dev' || profile.env === 'test' ? '-ctyun': ''
```

Comment:
What does the ctyun mean?

Reply:
Because of some problems in `-dev` api,  they give me this api by `-ctyun`, may be one cloud service provider

Reply:
I think it's better to config the whole HOST `https://cloudapigw-ctyun.open.rokid.com` in a file, this generating rule looks weird much.

Reply:
Yeah, it is weird much, but there are more weirdness like this in our homebase project, they are need to be optimized~

Reply:
There are many configs be hard coding in our homebase, they need to be optimized! I will optimize it in out restructure version.

Reply:
We should optimize something if we found it if this is not urgent too much.

Reply:
Okay, I modify it in core-cloud-logger and I will publish it, then reinstall this lib.